packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management.psd1

```diff
@@ -21,9 +21,9 @@
 
 
 @{
-ModuleVersion = '2.3.0'
+ModuleVersion = '3.0.0'
 
-GUID = 'dd9cad34-ad03-439b-b347-590625302c38'
+GUID = '2a3e34b4-5564-488e-aaf6-f2cba3f7f05d'
```

Comment:
So in our case the GUID means very little as this module is not 'published' for consumption like a more traditional module; The GUID is mainly used for other modules to pull a dependency on it, as long as the GUID is different between versions we're ok.  I think we could just inject a different GUID and version at build time.

Ultimately you could drop the PSD1 from the repo and generate the PSD1 entirely at Build Time, either through text manipulation via Maven or a PS script on a windows builder node.

Reference about the GUID in the manifest.
https://msdn.microsoft.com/en-us/library/dd878337(v=vs.85).aspx